evalml/pipelines/components/transformers/preprocessing/replace_nullable_types.py

```diff
@@ -0,0 +1,171 @@
+"""Transformer to replace features with the new nullable dtypes with a dtype that is compatible in EvalML."""
+from pandas.core import arrays as pca
+from woodwork import init_series
+from woodwork.exceptions import WoodworkNotInitError
+from woodwork.logical_types import (
+    BooleanNullable,
+    Categorical,
+    Double,
+    IntegerNullable,
+)
+
+from evalml.pipelines.components.transformers import Transformer
+from evalml.utils import infer_feature_types
+
+
+def is_nullable_int(series):
+    """Function to determine whether a column in a dataframe is a pandas NullableInteger type.
+
+    Args:
+        series: The series to detect NullableIntegers in.
+
+    Returns:
+        boolean: Whether the column was of NullableInteger type.
+
+    Raises:
+        AttributeError: If s
+    """
+    try:
+        is_pandas_nullable = isinstance(series.dtype, pca.integer.Int64Dtype)
+    except AttributeError:
+        is_pandas_nullable = False
+
+    try:
+        is_ww_nullable = isinstance(series.ww.logical_type, IntegerNullable)
```

Comment:
Woodwork has an `AgeNullable` logical type that will have the nullable integer pandas dtype, so it'd have `is_pandas_nullable == True` from line 29 but then get `is_ww_nullable` as False here. Would you want to include `AgeNullable` here as well, or should this just be for logical types that can be inferred by Woodwork?

(I recognize that in cases where the woodwork typing info matches the sereis, this won't be a problem, but if the typing info is no longer valid for the series, you may still want an `AgeNullable` logical type to be indicated here)

Reply:
Included!